shtuff.py

```diff
@@ -43,8 +131,10 @@ def resize():
     resize()
 
     def read_and_stuff_command():
+        cmd_file = get_cmd_file(os.getpid())
```

Comment:
If you find yourself passing args like this most of the time, try this:
```python
def get_cmd_file(pid=None):
    return os.path.join(CONFIG_DIR, f"{pid or os.getpid()}.command")
```

Reply:
I think it's better for the caller of `get_cmd_file` to have to explicitly think about the pid it's passing in here.